README.md

```diff
@@ -1,6 +1,6 @@
-# VSP Analytics
```

Comment:
Since we bumped the minimum manifest version, I think we should change it accordingly in README.md as well (line 9, outside of current change scope)